crypto/provider_core.c

```diff
@@ -1156,6 +1156,24 @@ static int provider_flush_store_cache(const OSSL_PROVIDER *prov)
     return 1;
 }
 
+static int provider_empty_store(const OSSL_PROVIDER *prov)
```

Comment:
The name is a little bit confusing to me. Maybe `provider_drop_from_store` ?

Reply:
Name change done